modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/facets/view/asset_entries.jsp

```diff
@@ -62,7 +62,7 @@ AssetEntriesSearchFacetDisplayContext assetEntriesSearchFacetDisplayContext = as
 				</li>
 
 				<%
-				for (AssetEntriesSearchFacetTermDisplayContext assetEntriesSearchFacetTermDisplayContext : assetEntriesSearchFacetDisplayContext.getTermDisplayContexts()) {
+				for (BucketDisplayContext assetEntriesSearchFacetTermDisplayContext : assetEntriesSearchFacetDisplayContext.getTermDisplayContexts()) {
```

Comment:
@gustavolimav, you should also change assetEntriesSearchFacetTermDisplayContext to bucketDisplayContext

Reply:
I see you did it later